pkg/datarepair/checker/checker.go

```diff
@@ -140,3 +142,19 @@ func lookupResponsesToNodes(responses *pb.LookupResponses) []*pb.Node {
 	}
 	return nodes
 }
+
+func dataAccounting(healthyNodeIds []string, pointer *pb.Pointer) {
+	pointersize := pointer.GetSize()
```

Comment:
Is this the total size of the data? Or the size of the pointer?

Reply:
If it's the size of the data, I'd change it to something like `segmentSize := pointer.GetSize()`
Then below, I'd change `nodeSize := pointerSize / int64(minReq)` to `pieceSize := segmentSize / int64(minReq)`
At least if I understand what's going on here.